ui/dashboards/src/components/DeletePanelDialog/DeletePanelDialog.tsx

```diff
@@ -53,15 +53,17 @@ const DeletePanelForm = ({ deletePanelDialog }: DeletePanelFormProps) => {
   };
   return (
     <form onSubmit={handleDelete}>
-      <DialogContent sx={{ width: '500px' }}>
+      <DialogContent dividers sx={{ width: '500px' }}>
```

Comment:
dialogs should have dividers and secondary button should have `outlined` variant